files/image_config/ntp/ntp.conf.j2

```diff
@@ -29,9 +29,13 @@ server {{ ntp_server }} iburst
 
 #only listen on localhost and eth0 ips (default is to listen on all ip addresses)
 interface ignore wildcard
+{% if MGMT_INTERFACE %}
```

Comment:
when will MGMT_INTERFACE be not defined? how to assume that eth0 is the one to use.

Reply:
For the first question, if the mgmt interface is not in the minigraph, then it will not be defined. I think we do have such cases for the default minigraph. It does not have management ip configured so dhcp is enabled.

for the second qeustion, it is a good question. but we are implicit assuming eth0 in several other places like the interface.j2. I think that is one of basic assumption here. This assumption can be relaxed later, but it is probably not related to this PR.

Reply:
@stcheng the changes made here is the similar how interfaces are interpreter int system https://github.com/Azure/sonic-buildimage/blob/master/files/image_config/interfaces/interfaces.j2#L23
MGMT_INTERFACE will not be defined when user will not add corresponding section in minigraph or configDB.

Reply:
thanks for the explanations!